### PR TITLE
Added React week3 homework files

### PR DESCRIPTION
**Pull request by sandhana14**

### CODE REVIEW

File: react/week3/todo-list/src/AddTodo.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+      {!validDate && <h3 className="error-tag">{error()}</h3>}
+
+      <div>
+        <button
+          onClick={() => {
+            addTodosAfterButtonClick(description, deadline);
+            setDescription("");
+            setDeadline("");
+          }}
```
Comment (obelitz) [inline react/week3/todo-list/src/AddTodo.js:71]: If you wanna move some of the logic out of the `<button>` tag, you can create a method called:
```
const onAddTodo = () => {
  addTodosAfterButtonClick(description, deadline);
  setDescription("");
  setDeadline("");
}
```

Then the onClick would simply be:
`onClick={onAddTodo}`

File: react/week3/todo-list/src/App.css [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  );
+}
+
+.display-todo:nth-child(4n + 4) {
+  background: linear-gradient(
+    90deg,
+    rgb(150, 145, 73) 0%,
+    rgb(201, 214, 11) 100%
+  );
+}
```
Comment (obelitz) [inline react/week3/todo-list/src/App.css:85]: Nice! ✨

File: react/week3/todo-list/src/RenderTodos.js [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import TodoItem from "./TodoItem";
+import Addtodo from "./AddTodo";
+import BorderComponent from "./BorderComponent";
+
+const RenderTodos = ({ todosArray }) => {
```
Comment (obelitz) [inline react/week3/todo-list/src/RenderTodos.js:6]: I feel like this component should be called `TodoList`

File: react/week3/todo-list/src/RenderTodos.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    setCurrentTodos(updatedTodos);
+  };
+
+  return (
+    <div>
+      <BorderComponent>
+        <Addtodo addTodosAfterButtonClick={addTodosAfterButtonClick} />
+      </BorderComponent>
+
+      {currentTodos.length !== 0 ? (
```
Comment (obelitz) [inline react/week3/todo-list/src/RenderTodos.js:47]: `0` is considered a `false` value so this check could be:
`{currentTodos.length ? (`

**Review by obelitz — COMMENTED** [review]
Good job!

File: react/week3/todo-list/src/RenderTodos.js (hunk 4 above)
Comment (sandhana14) [reply react/week3/todo-list/src/RenderTodos.js:47]: Thank you so much for your comments. I will make the changes as per your comments

**Pull request merged**